app/src/main/res/values/strings.xml

```diff
@@ -2,53 +2,53 @@
 <resources>
```

Comment:
Hmm, I don't think this is the correct way to implement translation.
Can you make a new values folder for Lithuanian and add this strings.xml into that?